addons/event/models/event_mail.py

```diff
@@ -118,11 +157,15 @@ def execute(self):
                     mail.write({'mail_registration_ids': lines})
                 # execute scheduler on registrations
                 mail.mail_registration_ids.filtered(lambda reg: reg.scheduled_date and reg.scheduled_date <= now).execute()
+                mail.sent_to_message = _('%i Attendees') % len(mail.mail_registration_ids.filtered(lambda reg: reg.mail_sent))
             else:
                 # Do not send emails if the mailing was scheduled before the event but the event is over
                 if not mail.mail_sent and (mail.interval_type != 'before_event' or mail.event_id.date_end > now) and mail.notification_type == 'mail':
                     mail.event_id.mail_attendees(mail.template_id.id)
                     mail.write({'mail_sent': True})
+
+                mail.sent_to_message = _('%i Attendees') % len(mail.event_id.registration_ids.filtered(lambda reg: reg.state != 'cancel'))
```

Comment:
Would be better to unstore and compute it à la volée. It is redundant information.

Reply:
@tde-banana-odoo same remark as above :/ can not be correctly computed if the `interval_type` is not `after_sub`

Reply:
I am sure we don't want to store that kind of information. It is useless storage use and this table suffers from too much concurrent issues.

Reply:
@tde-banana-odoo 
I can move back to my computed field, but if we add attendee after the mail sending (if the type is "after/before the event", the counter will be wrong :/ )